Cab share/Login/LoginView.swift

```diff
@@ -6,12 +6,27 @@
 //
 
 import SwiftUI
+import FirebaseCore
+import FirebaseAuth
 
 struct LoginView: View {
     @State private var email = ""
     @State private var password = ""
-    @State private var signIn = false
+    @State private var signInView = false
+    @State private var userIsLoggedIn = false
+    @State private var userSignedUp = false
+    @State private var userSignedIn = false
     var body: some View {
+        if userSignedIn {
+            MainView()
+        } else if userSignedUp {
+            UserView()
+        } else {
+            content
```

Comment:
Ahmed recommended we make this part
  var body: some View {
         if userSignedIn {
             MainView()
         } else if userSignedUp {
             UserView()
         } else {
             content

implement in a separate file because it's not a login view at all. It's just a part where we can understand whether the user is signedIn or not. And it's better to separate.

Reply:
yes, this should be moved into mainview